admin-guide/en/getting-started.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Getting Started with OJS, OMP or OPS - Admin Guide - PKP Developer Docs
+description: A quick-start guide to install OJS, OMP or OPS in a test environment.
+book: admin-guide
+version: 3.4
+---
+
+# Getting Started
+
+This is a quick-start guide to install and run OJS, OMP or OPS in a local test environment. Use this guide to try out the software on your own or to install it to a private, test environment. Before hosting the installation on a public server, you should read the [Deployment](./deploy) guide to learn how to safely configure the application.
+
+## System Requirements
+
+Your server — or your local development environment — must meet the following requirements in order to run the 3.4 release of OJS, OMP or OPS.
+
+- [PHP 8.0+](http://www.php.net/) with php-mbstring, php-xml, and php-intl enabled
+- [MySQL 4.1+](http://www.mysql.com/), [MariaDB 4.1+](https://mariadb.org/) or [PostgreSQL 9.5+](http://www.postgresql.org/)
+- Linux, or one of the following: BSD, Solaris, Mac OS X, Windows
+
+> **Older Versions**: Check the README.md file located in the download package of the version you want to install.
+
+## Download
+
+Download the latest stable release package (`.tar.gz`) from the PKP website.
+
+- [Open Journal Systems](https://pkp.sfu.ca/software/ojs/download) (OJS)
+- [Open Monograph Press](https://pkp.sfu.ca/software/omp/download) (OMP)
+- [Open Preprint Systems](https://pkp.sfu.ca/software/ops/download) (OPS)
+
+Use the `tar` command, or any archive software on your system, to unpack the `.tar.gz` file to a local directory. For example, if you downloaded the OJS 3.4.0 release package, you can use the following command to unpack the file.
+
+```
+tar -xvf ojs-3.4.0.tar.gz
+```
+
+This will create a directory for the software at `ojs-3.4.0`. Create another directory for file uploads.
+
+```
+mkdir ojs-files
```

Comment:
Add a warning that this needs to be outside the web root. (I know these are tryout instructions, but it's a very common mistake people make.)

Reply:
Ok, does this look alright? https://github.com/pkp/pkp-docs/pull/1091/commits/0b7fd3f451bc190a34f3a5f6d10801eb95e13095